### ISSUE TITLE
Unable to publish a package -- Publish hangs without errors

### ISSUE THREAD

**Issue by matheusgr** [jsr-io/jsr#875]
Package: https://github.com/deco-cx/deco (version 1.108.1)

Command line execution (it hangs on the last message):

```
matheusgr@decocx deco % deno publish          
Checking for slow types in the public API...

[...] a lot of warnings...

Visit https://jsr.io/auth?code=MPWD-SLSR to authorize publishing of @deco/deco
Waiting...
Authorization successful. Authenticated as Matheus Gaudencio do Rêgo
Publishing @deco/deco@1.108.1 ...
Already uploaded, waiting for publishing @deco/deco@1.108.1
```

Publish progress (from JSR page):
```
Waiting for some packages to complete publishing.
@deco/deco@1.108.1

Publishing has not started yet...
```

Dry run execute successfully. We have the same issue with our CI.

Task IDs (from jsr versions page):

```
Version: 1.108.1
Task ID: 9bff39dd-f7af-4a7d-bde1-d863fdb6ad2a
Created: 23 hours ago
Submitter: [View user](https://jsr.io/user/bf0dbdb3-77e0-481e-88ac-cb3889299523)
Status:
processing
```

**Comment by matheusgr** [comment]
It also happens to another user with publishing access (mcandeia).

**Comment by pcardune** [comment]
I think I'm running into this issue as well for the [@paul-db/core](https://jsr.io/@paul-db/core) package. A few days ago this completed in ~30 seconds. This is the first time I attempted to published something via a [github action](https://github.com/pcardune/paul-db/actions/runs/12436847039/job/34725493768), but I'm pretty sure my workflow configuration is correct.

<img width="426" alt="Screenshot 2024-12-20 at 11 14 11 AM" src="https://github.com/user-attachments/assets/19fbe252-048b-4971-ae6f-ee2a68b6ef13" />

**Comment by lucacasonato** [comment]
We are looking into it - it's related to a recent upgrade of our documentation generation system. Sorry.

**Comment by lucacasonato** [comment]
Fix: #877

**Comment by lucacasonato** [comment]
This is fixed and all affected publishes are now unstuck and have succeeded
